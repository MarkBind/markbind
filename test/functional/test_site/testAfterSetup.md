<frontmatter>
    layout: testAfterSetup
</frontmatter>

<box id="alert-box">
    This Vue element should trigger a alert dialog when clicked on.
</box>

<span id="mouseover-span">This span should toggle the colour of the above box when hovered upon.</span>
