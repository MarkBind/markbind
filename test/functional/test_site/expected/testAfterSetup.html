<!DOCTYPE html>
<html>
<head>
    <meta name="head-top">
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Hello World</title>
    <link rel="stylesheet" href="markbind/css/bootstrap.min.css">
    <link rel="stylesheet" href="markbind/css/bootstrap-vue.min.css">
    <link rel="stylesheet" href="markbind/css/font-awesome.min.css" >
    <link rel="stylesheet" href="markbind/css/bootstrap-glyphicons.min.css" >
    <link rel="stylesheet" href="markbind/css/github.min.css">
    <link rel="stylesheet" href="markbind/css/markbind.css">
    <link rel="stylesheet" href="markbind/layouts/testAfterSetup/styles.css">
    
    
    <meta name="head-bottom">
    <link rel="icon" href="/test_site/favicon.png">
</head>
<body>
<div id="loading-overlay">
    <div id="spinner-layout">
        <div id="spinner"></div>
    </div>
</div>
<script type="text/javascript">
    document.getElementById("loading-overlay").style.display = 'initial';
</script>
<div id="app">
    <div id="content-wrapper">
  <box id="alert-box">
    This Vue element should trigger a alert dialog when clicked on.
  </box>
  <p><span id="mouseover-span">This span should toggle the colour of the above box when hovered upon.</span></p>
</div>
<div id="flex-div"></div>
<footer>
  <div class="text-center">
    testAfterSetup footer
  </div>
</footer>
</div>
</body>
<script src="markbind/js/vue.min.js"></script>
<script src="markbind/js/vue-strap.min.js"></script>
<script src="markbind/js/bootstrap-utility.min.js"></script>
<script src="markbind/js/polyfill.min.js"></script>
<script src="markbind/js/bootstrap-vue.min.js"></script>
<script>
    const baseUrl = '/test_site'
    const enableSearch = true
</script>
<script src="markbind/js/setup.js"></script>
<script src="markbind/layouts/testAfterSetup/scripts.js"></script>
</html>
